Runtime/Shapes/Shape.cs

```diff
@@ -35,6 +35,8 @@ public virtual Bounds UpdateBounds(ProBuilderMesh mesh, Vector3 size, Quaternion
         /// </summary>
         /// <param name="shape">The <see cref="Shape" /> to copy property values from.</param>
         public abstract void CopyShape(Shape shape);
+
+        internal abstract void SetParametersToBuiltInShape();
```

Comment:
Just a detail here, I would maybe think that `ResetDefaultShapeParameters` is a little more explicit here ?

Reply:
Whatever, ignore this